react/src/pages/EndpointDetailPage.tsx

```diff
@@ -522,6 +609,203 @@
           items={items}
         ></Descriptions>
       </Card>
+      <Card
+        title={t('modelService.AutoScalingRules')}
+        extra={
+          <Button
+            type="primary"
+            icon={<PlusOutlined />}
+            disabled={isDestroyingStatus(
+              endpoint?.replicas ?? endpoint?.desired_session_count,
+              endpoint?.status,
+            )}
+            onClick={() => {
+              setIsOpenAutoScalingRuleModal(true);
+            }}
+          >
+            {t('modelService.AddRules')}
+          </Button>
+        }
+      >
+        <Table
+          scroll={{ x: 'max-content' }}
+          rowKey={'id'}
+          columns={[
+            {
+              title: '#',
+              fixed: 'left',
+              render: (id, record, index) => {
+                ++index;
+                return index;
+              },
+              showSorterTooltip: false,
+            },
+            {
+              title: 'Metric Name',
+              dataIndex: 'metric_name',
+              fixed: 'left',
+              render: (text, row) => (
+                <Typography.Text ellipsis copyable style={{ width: 150 }}>
+                  {row?.metric_name}
+                </Typography.Text>
+              ),
+            },
+            {
+              title: t('modelService.Controls'),
+              dataIndex: 'controls',
+              key: 'controls',
+              render: (text, row) => (
+                <Flex direction="row" align="stretch">
+                  <Button
+                    type="text"
+                    icon={<SettingOutlined />}
+                    loading={isPendingReload}
```

Comment:
Personally, all control buttons enter a loading state whenever an item is added or removed, which might appear awkward. How about deleting this?